pkg/storage/storagewrappers/check_caching.go

```diff
@@ -59,6 +59,12 @@ var (
 		NativeHistogramMaxBucketNumber:  100,
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"operation"})
+
+	tuplesInvalidCacheHit = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      "tuples_iterator_cache_invalid_hit_count",
+		Help:      "The total number of times we hit an invalid tuple iterator cache entry.",
+	})
```

Comment:
```suggestion
	tuplesInvalidCacheHit = promauto.NewCounter(prometheus.CounterOpts{
		Namespace: build.ProjectName,
		Name:      "tuples_iterator_cache_invalid_hit_count",
		Help:      "The total number of cache hits for a cached iterator that were discarded because they were invalidated",
	})
```